Payload/bleemsync/bin/execute

```diff
@@ -8,42 +8,39 @@
 ini_file="/media/bleemsync/etc/bleemsync/CFG/bleemsync_cfg.INI"
 cfg_file="/var/volatile/bleemsync.cfg"
 
-[ -f "$cfg_file" ] && rm -f "$cfg_file" && touch "$cfg_file"
-echo "#!/bin/sh" >> "$cfg_file"
-cat "$ini_file" | grep -v ";" | grep -v "\[" >> "$cfg_file"
+rm -f "$cfg_file"
```

Comment:
Don't need to check whether a file exists when you're using `rm -f`. In fact, we don't really need to `rm` it at all since now it's a single operation and overwritten.